### ISSUE TITLE
Can't run maven in eclipse

### ISSUE THREAD

**Issue by pkrall520** [pmd/pmd-eclipse-plugin#38]
Hello, 
I pulled the latest master with the checkstyle code and it is failing when I import it into eclipse and run a maven update. 

Eclipse yells at me and tells me that it can't find the tools jar from the jar. I have fixed this by updating the checkstyle dependency as follows

```
<dependencies>
    <dependency>
        <groupId>com.puppycrawl.tools</groupId>
        <artifactId>checkstyle</artifactId>
        <version>8.8</version>
        <exclusions>
            <exclusion>
                <artifactId>tools</artifactId>
                <groupId>com.sun</groupId>
            </exclusion>
        </exclusions>
    </dependency>
</dependencies>
```

**Comment by adangel** [comment]
Hm...that's strange. Which JDK do you have installed? The eclipse plugin currently is building for Java 1.6 (which is wrong and will shortly be updated to at least Java 1.7 to match PMD). As long as you use Java 1.7 or 1.8, it *should* work. With Java9, it also *should* work. I just tried it, all version work for me. Java 9 doesn't have the tools.jar anymore, so the error should only appear with Java 9, if at all. I think, it has been fixed in checkstyle itself with https://github.com/checkstyle/checkstyle/issues/2905 (Checkstyle 6.16 and later).

Which checkstyle eclipse plugin version do you have? I'm using   8.8.0.201802200627 (Help -> About Eclipse -> Installation Details)

**Comment by pkrall520** [comment]
Eclipse Version: Oxygen.2 Release (4.7.2)
Eclipse Checkstyle Plug-in 8.8.0.201802200627

I have a bunch of java versions since my company supports 6-10.
I am currently running the plugin with jdk1.8.0_131

Also, my to PRs failed because it doesn't like the order of the imports.
The imports are organized by eclipse and seem to be in alphabetical order

import name.herlin.command.CommandException;
import name.herlin.command.Timer;
import net.sourceforge.pmd.Rule;
import net.sourceforge.pmd.RuleSet;    <- Problem with this guy
import net.sourceforge.pmd.eclipse.plugin.PMDPlugin;
import net.sourceforge.pmd.eclipse.runtime.PMDRuntimeConstants;
import net.sourceforge.pmd.eclipse.runtime.builder.MarkerUtil;
import net.sourceforge.pmd.eclipse.runtime.preferences.IPreferences;
import net.sourceforge.pmd.eclipse.runtime.properties.IProjectProperties;
import net.sourceforge.pmd.eclipse.runtime.properties.PropertiesException;
import net.sourceforge.pmd.eclipse.ui.actions.RuleSetUtil;
import net.sourceforge.pmd.util.StringUtil;



The issue about tools and now this odd imports issues leads me to believe
that it is an issue with checkstyles and/or some off configuration on my
machine


Phillip Krall

**Comment by adangel** [comment]
I've just pushed my latests changes - there should be no java error anymore.

I did not encounter your checkstyle problem so far. But I only ran "Maven -> Update Project..." (instead of reimporting).

I'm running Oxygen.3 (4.7.3) (not sure, if this matters).

About the import order problem, make sure to load the configuration file [pmd-eclipse-imports.importorder](https://github.com/pmd/build-tools/blob/master/config/eclipse/pmd-eclipse-imports.importorder) as the workspace default or for each project.
